src/install/db.py

```diff
@@ -39,8 +43,24 @@ def postgres_is_installed():
         return False
 
 
+def postgres_is_up_to_date():
+    with suppress(CalledProcessError, FileNotFoundError):
+        proc = run(split('psql --version'), text=True, capture_output=True, check=False)
```

Comment:
Ok. I get that this is super nitpicky, but there is really no need to call `psql --version` twice :wink:

Reply:
removed one call